applications/web/templates/ingress.yaml

```diff
@@ -1,9 +1,12 @@
 {{- if .Values.ingress.enabled -}}
 {{- $fullName := include "docker-template.fullname" . -}}
 {{- $svcPort := .Values.service.port -}}
+{{- $allHosts := concat .Values.ingress.hosts, .Values.ingress.porter_hosts -}}
 {{- $customPaths := .Values.ingress.custom_paths -}}
-{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
-apiVersion: networking.k8s.io/v1beta1
+{{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1" }}
+apiVersion: networking.k8s.io/v1
+{{- else if .Capabilities.APIVersions.Has "networking.k8s.io/v1beta" -}}
+apiVersion: networking.k8s.io/v1beta
 {{- else -}}
 apiVersion: extensions/v1beta1
```

Comment:
was this removed in v1.14? when should we remove this from the chart definition? :) 
I'm just keeping it as a defense mechanism but would love to learn of what our thoughts are regarding this.